pom.xml

```diff
@@ -101,6 +101,11 @@
             <artifactId>maven-compiler-plugin</artifactId>
             <version>3.8.1</version>
         </dependency>
+        <dependency>
```

Comment:
I think this is a pretty small dependency. We might consider just using an xml reader though.

Reply:
You might even be able to read the file into a string and do a regex to find the version without needing to actually parse it. Though that might be a little more brittle, but it would avoid having to pull in another dependency.